autocertkit/utils.py

```diff
@@ -1914,54 +1908,65 @@ def __call__(self, *args):
             signal.signal(signal.SIGALRM, old)
         return result
 
-def xml_to_dicts(xml, tag):
-    doc = minidom.parseString(xml.strip())
-    try:
-        result = []
-        elems = doc.getElementsByTagName(tag)
-        for el in elems:
-            # Note that we have to convert this dictionary to non-unicode
-            # strings, because we're being casual elsewhere.  That's why we're
-            # not just returning dict(el.attributes.items()).
-            item = {}
-            for k, v in el.attributes.items():
-                item[str(k)] = str(v)
-            result.append(item)
-    finally:
-        doc.unlink()
-    log.debug(result)
-    return result
+def json_loads(json_data):
```

Comment:
I am still not convinced that we need this method. 
# Why space in the key problem?
# Why do we need to parse when value is list or dict?
# What is source of unicode? I thought we all use utf-8 everywhere in XS.